cms/envs/common.py

```diff
@@ -2500,9 +2500,9 @@
     # STORAGE_KWARGS=dict(bucket='video-image-bucket'),
     STORAGE_KWARGS=dict(
         location=MEDIA_ROOT,
-        base_url=MEDIA_URL,
     ),
     DIRECTORY_PREFIX='video-images/',
+    BASE_URL=MEDIA_URL,
```

Comment:
https://github.com/openedx/edx-platform/pull/20901

Reply:
These values override for sandbox and stage/prod and there base_url does not exists.  Its for devstack.

Reply:
These are default settings as per [base.py](https://github.com/jschneier/django-storages/blob/1.10.1/storages/base.py#L5)
```
{'access_key': '**', 'secret_key': '**', 'file_overwrite': True, 'object_parameters': {}, 'bucket_name': 'awaisqureshi', 'querystring_auth': True, 'querystring_expire': 3600, 'signature_version': None, 'location': '', 'custom_domain': None, 'cloudfront_signer': None, 'addressing_style': None, 'secure_urls': True, 'file_name_charset': 'utf-8', 'gzip': False, 'gzip_content_types': ('text/css', 'text/javascript', 'application/javascript', 'application/x-javascript', 'image/svg+xml'), 'url_protocol': 'http:', 'endpoint_url': None, 'proxies': None, 'region_name': None, 'use_ssl': True, 'verify': None, 'max_memory_size': 0, 'default_acl': None}
```
If pass any param to constructor it verifies all those params against above mention params and throws error if its not exists in defaults.